Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -118,7 +118,7 @@ static void IPC_HLE_UpdateCallback(u64 userdata, int cyclesLate)
 static void VICallback(u64 userdata, int cyclesLate)
 {
 	VideoInterface::Update();
-	CoreTiming::ScheduleEvent(VideoInterface::GetTicksPerHalfLine() - cyclesLate, et_VI);
+	CoreTiming::ScheduleEvent(s64(VideoInterface::GetTicksPerHalfLine()) - cyclesLate, et_VI);
```

Comment:
It might be worthwhile considering to change the signature of (or add an overload to) `ScheduleEvent` that takes an `s64` parameter; since this is probably not the only instance where such a wraparound/promotion happens.